apps/payments/.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd="pg_isready" --health-interval=10s --health-timeout=5s --health-retries=3
+
+      # redis:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)